adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/TokenCache.cs

```diff
@@ -68,6 +68,9 @@ static TokenCache()
                 BeforeAccess = StorageDelegates.BeforeAccess,
                 AfterAccess = StorageDelegates.AfterAccess
             };
+
+            if (CoreLoggerBase.Default == null)
+                CoreLoggerBase.Default = new AdalLogger(Guid.Empty);
```

Comment:
should be enough?

Reply:
@henrik-me For the Automation, yes, this is enough. The others were just to ensure all bases are covered.